addons/website/static/lib/summernote/src/js/EventHandler.js

```diff
@@ -420,7 +420,7 @@ define([
           $btn.parents('.popover').hide();
         }
 
-        if ($.isFunction($.summernote.pluginEvents[eventName])) {
+        if ($.isFunction($.summernote.pluginEvents[eventName]) && eventName != "formatBlock") {
```

Comment:
Ok, I revert this changes and make another task for fix code, quote formatting plugin.